src/monitor-canister/src/periodic.rs

```diff
@@ -105,7 +105,7 @@ fn make_get_exchange_rate_requests(timestamp: u64) -> Vec<GetExchangeRateRequest
                 class: AssetClass::Cryptocurrency,
             },
             Asset {
-                symbol: "BTT".to_string(),
+                symbol: "USDT".to_string(),
```

Comment:
Why change the symbol?
Note that BTT is used for a reason: The BTC/BTT rate is a good test case because the BTT token has a very small value.
Note that the comment above has not been updated yet (if the symbol should indeed be changed).

Reply:
@THLO That was a mistake to leave those changes in. Thank you for catching them. I have reverted them and the beta XRC is handling them properly.